.github/workflows/python-lint.yml

```
@@ -0,0 +1,17 @@
+name: Python lint
+on: 
+  pull_request:
+    paths:
+      - '.github/workflows/python-lint.yml'
+      - 'api/**'
+      - 'apis/**'
+      - 'framework/**'
+      - 'integrations/**'
+      - 'wodles/**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest 
```

We should define a specific version instead of using the latest to avoid future compatibility issues.
```suggestion
    runs-on: ubuntu-24.04
```